rocky/tools/models.py

```diff
@@ -155,13 +155,13 @@ class OrganizationMember(models.Model):
     class STATUSES(models.TextChoices):
```

Comment:
Do we still need this? What's the difference between 'active' and 'new'?

Reply:
I see! Maybe this should also be a Boolean value instead?

Reply:
That could be, but not necessarily imo. I can't foresee if there will be more statuses added in the future. My suggestion would be to leave this as is for now.

Reply:
Ok!